### ISSUE TITLE
missing sandbox module

### ISSUE THREAD

**Issue by TheGoldenEye** [LiskHQ/lisk#76]
lisk v0.2.0:
I get the following error if I'm starting the client:
```
domain master { message: 'No such module: sandbox',
  stack: 'Error: No such module: sandbox\n    at Error (native)\n    at Array.async.auto.sandbox (/home/jens/src/coins/lisk/binary/lisk-0.2.0-Linux-x86_64/dapps/10113539455696385153/index.js:24:21)\n    at /home/jens/src/coins/lisk/binary/lisk-0.2.0-Linux-x86_64/dapps/10113539455696385153/node_modules/async/lib/async.js:596:38\n    at _arrayEach (/home/jens/src/coins/lisk/binary/lisk-0.2.0-Linux-x86_64/dapps/10113539455696385153/node_modules/async/lib/async.js:85:13)\n    at Object.async.auto (/home/jens/src/coins/lisk/binary/lisk-0.2.0-Linux-x86_64/dapps/10113539455696385153/node_modules/async/lib/async.js:554:9)\n    at Domain.<anonymous> (/home/jens/src/coins/lisk/binary/lisk-0.2.0-Linux-x86_64/dapps/10113539455696385153/index.js:22:8)\n    at Domain.run (domain.js:234:16)\n    at Object.<anonymous> (/home/jens/src/coins/lisk/binary/lisk-0.2.0-Linux-x86_64/dapps/10113539455696385153/index.js:21:3)\n    at Module._compile (module.js:460:26)\n    at Object.Module._extensions..js (module.js:478:10)' }
```
npm install in the node_modules/lisk-sandbox directory was not successful

**Comment by karmacoma** [comment]
@TheGoldenEye It's working fine here. Maybe do a `rm -rf node_modules/lisk_sandbox` and try `npm install` again.

**Comment by TheGoldenEye** [comment]
Same as before. After deleting the lisk-sandbox folder I re-created it using `npm install lisk-sandbox` from the lisk root directory.  `npm install` was not successful.
Unfortunately no change, the error is the same.

**Comment by TheGoldenEye** [comment]
The problem only occurs when you try to start a dapp

**Comment by TheGoldenEye** [comment]
Sandbox is working now. No idea, why :-O

**Comment by TheGoldenEye** [comment]
I reopened the issue because I have new hints.
If I use the node from 28-04-2016 (v0.12.13) (https://downloads.lisk.io/lisk-node/lisk-node-Linux-x86_64.tar.gz) I get the sandbox error.
Currently I use an older version (03-02-2016, v0.12.9) without problems.

Perhaps it is relevant: My locally installed node version is v0.12.14.

**Comment by karmacoma** [comment]
@TheGoldenEye I will investigate further, as it could be a bug introduced when moving to the newer version of node.

**Comment by karmacoma** [comment]
This issue applies to lisk-node / lisk-build. It looks like the required module was not  included properly in the last build. My fault most likely. I will fix now.

**Comment by karmacoma** [comment]
Will bump the version to 0.12.14 while I'm at it.

**Comment by karmacoma** [comment]
Issue now resolved for version 0.2.3 at: https://github.com/LiskHQ/lisk-node.
